### DISCUSSION TITLE
The shape of the output is not the same as the original shape of my data--UNETR

### DISCUSSION THREAD

**Discussion by daisysong1** [Project-MONAI/tutorials#1265] [Q&A]
I checked that the input in the validation was already a miniaturized shape.
I wanna know if this has anything to do with transform.
What changes should I make if I want to end up with the original shape ?
If anyone can help me, thank you very much!

**Comment by KumoLiu** [comment]
Hi @daisysong1, did you follow this tutorial? https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/unetr_btcv_segmentation_3d.ipynb
If so, removing `Spacingd` and `CropForegroundd` can help you keep data after `val_transforms` with the original shape.

BTW, if you remove `Spacingd` and `CropForegroundd` in the validation, also update the `train_transforms` to avoid performance drop due to differences between the training and the validation dataset.

Hope it can help you, thanks!

**Comment by daisysong1** [comment]
Thank you very much for your reply. I'll train again and see how accurate it is.
